.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,26 +1,29 @@
 ---
 name: 🐞 Bug Report
-about: Report a bug to help improve Self-Hosted Sentry
+about: Report a bug in Self-Hosted Sentry
 ---
 
-## Version Information
+### Version Information
```

Comment:
Should all these be `##` instead?

Reply:
That was not my intention but I can go back to h2.

Reply:
Alright, let's use `h3` then. Thanks for this fix :)